src/lib/services/nft-collection/index.ts

```diff
@@ -102,25 +91,31 @@ export const useCollectionByCollectionAddress = (
 };
 
 export const useCollectionCreator = (collectionAddress: HexAddr32) => {
-  const { chainConfig } = useCelatoneApp();
   const {
     chain: { bech32_prefix: prefix },
   } = useCurrentChain();
   const { tier } = useTierConfig();
+  const apiEndpoint = useBaseApiRoute("nft_collections");
   const lcdEndpoint = useLcdEndpoint();
 
   return useQuery<CollectionCreatorResponse>(
     [
       CELATONE_QUERY_KEYS.NFT_COLLECTION_CREATOR,
-      chainConfig.graphql,
       collectionAddress,
+      prefix,
+      tier,
+      apiEndpoint,
+      lcdEndpoint,
     ],
```

Comment:
reorder to `[CELATONE_QUERY_KEYS.NFT_COLLECTION_CREATOR, apiEndpoint, lcdEndpoint, tier, prefix, collectionAddresstier]`